test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/TestApplication/TestApplicationFactory.cs

```diff
@@ -134,13 +127,17 @@ private static WebApplication CreateMinimalApiApplication()
         app.Urls.Clear();
         app.Urls.Add("http://[::1]:0");
 
+#if NET7_0_OR_GREATER
```

Comment:
Another minor one: might be nice to have this with and without MapGroup in case the behavior differs somehow

Reply:
Yea fair enough. Testing both now.